operator/v1/types.go

```diff
@@ -135,6 +135,8 @@ var (
 	OperatorStatusTypeProgressing = "Progressing"
 	// Failing indicates that the operator (not the operand) is unable to fulfill the user intent
```

Comment:
Does the Failing type comment need to be updated since it's the same as Degraded?